ui/src/app/base/components/NetworkTab/NetworkTab.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { mount } from "enzyme";
+
+import NetworkTab, { ExpandedState } from "./NetworkTab";
```

Comment:
This is minor, but I feel like `NodeNetworkTab` might be a more indicative component name. `NetworkTab` seems like it could belong in the subnets section of the app.

Reply:
Good point, thanks.